modular_skyrat/modules/wirebrush_rust/code/game/objects/items/wirebrush.dm

```diff
@@ -0,0 +1,45 @@
+/obj/item/wirebrush
+	name = "wirebrush"
+	desc = "A tool that is used to scrub the rust thoroughly off walls. Not for hair or fur!"
+	icon = 'modular_skyrat/modules/wirebrush_rust/icons/obj/wirebrush.dmi'
+	icon_state = "wirebrush"
+	///variable that determines how long it takes to "clean" (read: replace(?)) rusted walls.
+	var/clean_speed = 30 //3 seconds
+
+/obj/item/wirebrush/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
```

Comment:
optionally, you could make this work with the cleaning skill similar to other janitor tools! optional tho
```suggestion
	. = ..()
	if(user.mind)
		cleanspeed = cleanspeed * min(user.mind.get_skill_modifier(/datum/skill/cleaning, SKILL_SPEED_MODIFIER)+0.1,1)
```